adapters/repos/db/shard_lazyloader.go

```diff
@@ -75,9 +80,11 @@ type deferredShardOpts struct {
 }
 
 func (l *LazyLoadShard) MustLoad() {
+	/* Sigh
 	if l.loaded {
-		return
+		return nil
 	}
+	*/
```

Comment:
Remove?

Reply:
Not sure if I should do a read lock here or what.  The fact that go doesn't have lock-free concurrency is very annoying here.

Reply:
Either read lock, or atomic bool would probably be fine